db/seeds.rb

```diff
@@ -6,14 +6,12 @@
 if Rails.env.development?
   # Create a few dummy accounts for use during development.  These accounts all
   # have 'password' as password and are setup for mobile OTP delivery.
-  mobile = 4_155_555_555
-  %w(test1@test.com test2@test.com).each do |email|
+  %w(test1@test.com test2@test.com).each_with_index do |email, index|
     User.find_or_create_by!(email: email) do |user|
       user.skip_confirmation!
       user.reset_password('password', 'password')
-      user.unconfirmed_mobile = mobile.to_s
+      user.unconfirmed_mobile = format('+1 (415) 555-%04d', index)
```

Comment:
Apparently, not all 555 numbers are unassigned. Only 555-0100 through 555-0199 according to Wikipedia: https://en.wikipedia.org/wiki/555_(telephone_number)

Could we update this accordingly just to avoid accidental spamming of real numbers if a developer is testing live SMS sending?